services/api/src/resources/user/resolvers.ts

```diff
@@ -246,9 +247,16 @@ export const addUserToOrganization: ResolverFn = async (
     });
     updateUser.owner = true
   } else {
-    await hasPermission('organization', 'addViewer', {
-      organization: organization
-    });
+    if (admin) {
+      await hasPermission('organization', 'addOwner', {
+        organization: organization
+      });
+      updateUser.admin = true
+    } else {
```

Comment:
This isn't being set - so admins don't get the admin bit set (but owners seemingly do?
```
{
	"data": {
		"allOrganizations": [
			{
				"id": 1,
				"name": "lagoon-demo-organization",
				"projects": [
					{
						"name": "lagoon-demo-org"
					}
				],
				"owners": [
					{
						"email": "orgowner@example.com",
						"owner": true,
						"admin": true
					},
					{
						"email": "orgviewer@example.com",
						"owner": null,
						"admin": null
					},
					{
						"email": "orgadmin@example.com",
						"owner": null,
						"admin": null
					}
				]
			}
		]
	}
}
```

Reply:
:eyes:

Reply:
[this is the fix](https://github.com/uselagoon/lagoon/pull/3733/files#diff-9601ce160bccb3faab48a4a65bd13ff8250a9e35949b996d62ff08fe33f8218eR323), was a copypasta error before where it was adding it to `filteredOwners` instead of `filteredAdmins`